<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoUpdaterDialog</class>
 <widget class="QDialog" name="AutoUpdaterDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Automatic Updater</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources/resources.qrc">:/icons/update.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Update Available</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="currentVersion">
     <property name="text">
      <string>Current Version: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="newVersion">
     <property name="text">
      <string>New Version: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="downloadSize">
     <property name="text">
      <string>Download Size: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="updateNotes"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="downloadAndInstall">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Download and Install...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="skipThisUpdate">
       <property name="text">
        <string>Skip This Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remindMeLater">
       <property name="text">
        <string>Remind Me Later</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>updateNotes</tabstop>
  <tabstop>downloadAndInstall</tabstop>
  <tabstop>skipThisUpdate</tabstop>
  <tabstop>remindMeLater</tabstop>
 </tabstops>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
